.env

```diff
@@ -5,4 +5,4 @@ REACT_APP_CHAIN_2_CUSTOM_TYPES_URL=https://raw.githubusercontent.com/paritytech/
 REACT_APP_CHAIN_2_SUBSTRATE_PROVIDER=wss://wss.millau.brucke.link
 
 REACT_APP_LANE_ID=0x00000000
-REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS=false
\ No newline at end of file
+REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS=false
```

Comment:
please remove this line.

Reply:
I did not add this line - why would I remove it?

Reply:
If I do the app want start and an error will appear:
```
Error: Missing REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS variable
```